_tuning-your-cluster/availability-and-recovery/remote-store/remote-cluster-state.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Remote Cluster State
+nav_order: 5
+parent: Remote-backed storage
+grand_parent: Availability and recovery
+---
+
+# Remote cluster state
+
+Introduced 2.10
+{: .label .label-purple }
+
+The _Remote Cluster State_ functionality for remote-backed storage, protects against any cluster state metadata loss resulting due to quorum loss, permanently losing the majority of cluster manager nodes, inside the cluster.
+
+_Cluster State_ is an internal data structure which contains the metadata of the cluster, including: 
+- Index settings 
+- Index mappings 
+- Active copies of shards in the cluster 
+- Cluster level settings
+- Data streams
+- Templates
+Cluster durability is the function of the cluster manager node's storage. The cluster state metadata is managed by the elected cluster manager node and is essential for the cluster to properly function. When the cluster loses the majority of the cluster manager nodes permanently, then the cluster can experience data loss because the latest cluster state metadata might not be present in the surviving cluster manager nodes. Persisting the state of all the cluster manager nodes in the cluster to remote back-storage provides better durability.
```

Comment:
```suggestion
The cluster state metadata is managed by the elected cluster manager node and is essential for the cluster to properly function. When the cluster loses the majority of the cluster manager nodes permanently, then the cluster may experience data loss because the latest cluster state metadata might not be present in the surviving cluster manager nodes. Persisting the state of all the cluster manager nodes in the cluster to remote-backed storage provides better durability.
```